.nuspec/Microsoft.Maui.Controls.SingleProject.props

```diff
@@ -35,7 +35,7 @@
         IsDefaultItem="true"
     />
     <ImageAsset
-        Include="$(iOSProjectFolder)**/*.xcassets/**/*.png;$(iOSProjectFolder)**/*.xcassets/*/*.jpg;$(iOSProjectFolder)**/*.xcassets/**/*.pdf;$(iOSProjectFolder)**/*.xcassets/**/*.json"
+        Include="$(iOSProjectFolder)**/*.xcassets/**/*.png;$(iOSProjectFolder)**/*.xcassets/*/*.jpg;$(iOSProjectFolder)**/*.xcassets/**/*.pdf;$(iOSProjectFolder)**/*.xcassets/**/*.svg;$(iOSProjectFolder)**/*.xcassets/**/*.json"
```

Comment:
@rolfbjarne I was looking at what is done in the iOS targets:

https://github.com/xamarin/xamarin-macios/blob/352a0d818a3ec8d7a7dc754d96150130b7aae62e/dotnet/targets/Microsoft.Sdk.DefaultItems.template.props#L36

Should we actually be doing `**\*.xcassets\**\*.*` here instead and not listing file extensions?

Reply:
I think it would make sense to not list the file extensions, and do the same as we do in iOS.

We did some testing, and it turned out you can put anything inside an *.xcassets directory (https://github.com/xamarin/xamarin-macios/commit/a0ad207ea7962be38ae02c8b96a320ae898ad4c0).

Reply:
I haven't opened a ticket for this, but upon testing resources in main assemblies vs sub assembles it turns out iOS is overwriting main assembly resources with sub assembly resources. Android (correctly?) layers them together. So, if I have a drawable foo in sub assembly and bar in main assembly, the app sees both foo and bar. If I have the same in .xcassets then the app only sees foo.

Reply:
Could you open a new issue on that against the macios repo?